src/stars.css

```
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 > .stars__star:nth-child(-n+1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 > .stars__star:nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 > .stars__star:nth-child(-n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 > .stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 > .stars__star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
```

Тобі про це писали раніше чому не виправив?

```suggestion
.stars--1 > .stars__star:nth-child(-n+1),
.stars--2 > .stars__star:nth-child(-n+2),
.stars--3 > .stars__star:nth-child(-n+3),
.stars--4 > .stars__star:nth-child(-n+4),
.stars--5 > .stars__star:nth-child(-n+5),
  background-image: url(./images/star-active.svg);
}
```